vignettes/ci.Rmd

```diff
@@ -97,15 +97,12 @@ The following template can be used as a base when using renv with
 
 ```
 variables:
-  RENV_CONFIG_REPOS_OVERRIDE: "http://cran.r-project.org"
-  RENV_PATHS_CACHE: ${CI_PROJECT_DIR}/cache
-  RENV_PATHS_LIBRARY: ${CI_PROJECT_DIR}/renv/library
```

Comment:
Specifying `/library` warned that this path did not exist. Removing this specification still preserved the expected caching behavior:
1. First run: ~4 min, created cache
2. Second run: ~2 min, leveraged all cached packages via `renv::restore()`